requirements.txt

```diff
@@ -24,7 +24,9 @@ amqp==2.2.2
 vine==1.1.4
 itsdangerous==1.1.0
 lxml==4.6.2
-mailchimp==2.0.9
+#mailchimp==2.0.9  deal with conflict later
+git+git://github.com/Johnetordoff/mailchimp-api-python.git
```

Comment:
Frustratingly mailchimp and internet archive version of docutils conflicts... and mailchimp doesn't appear to actually use docutils at all... so I had to fork it and remove the requirement.